src/load.c

```diff
@@ -221,8 +247,8 @@ main(argc, argv)
 			       "INSERT INTO warehouse values(?,?,?,?,?,?,?,?,?)",
 			       47) ) goto Error_SqlCall_close;
 	if( mytnt_stmt_prepare(stmt[2],
-			       "INSERT INTO stock values(?,?,?,?,?,?,?,?,?,?,?,?,?,0,0,0,?)",
-			       59) ) goto Error_SqlCall_close;
+			       "INSERT INTO stock values(?,?,?,?,?,?,?,?,?,?,?,?,?,NULL,0,0,?)",
+			       62) ) goto Error_SqlCall_close;
```

Comment:
I don't see anything that would advocate the change neither in the schema changes, nor in the commit message. Please, clarify.